client/js/render.js

```diff
@@ -145,22 +145,10 @@ function renderChannelMessages(data) {
 
 function renderChannelUsers(data) {
 	const users = chat.find("#chan-" + data.id).find(".users");
-	let nicks = users.data("nicks") || [];
-	const oldSortOrder = {};
-
-	for (const i in nicks) {
-		oldSortOrder[nicks[i]] = i;
-	}
-
-	nicks = [];
-
-	for (const i in data.users) {
-		nicks.push(data.users[i].nick);
-	}
-
-	nicks = nicks.sort(function(a, b) {
-		return (oldSortOrder[a] || Number.MAX_VALUE) - (oldSortOrder[b] || Number.MAX_VALUE);
-	});
+	const nicks = data.users
+		.concat()
```

Comment:
Oh. Well that's a very confusing way of doing it. I thin kthe standard way to do it is with slice(0), isn't it?